src/applications/hca/enrollment-status-helpers.jsx

```diff
@@ -357,10 +357,10 @@ export function getFAQContent(enrollmentStatus) {
 
   const faqBlock1 = (
     <>
-      <h4>
+      <h3>
```

Comment:
Just want to confirm as I haven't tested this locally. Is the styling still the same after upgrading these `h4` tags to `h3`?

Reply:
You could add something like this to ensure that the `h3` tags still have the `h4` tag styling:

```jsx
<h3 className="vads-u-font-size--h4"></h3>
```